WSL/troubleshooting.md

```diff
@@ -247,3 +247,19 @@ This error is related to being in a bad install state. Please complete the follo
 - Update your version of Windows by going to Settings, Updates, and clicking 'Check for Updates'
 
 - If both of those fail and you need to access WSL please consider upgrading in place by reinstalling Windows 10 using installation media and selecting 'Keep Everything' to ensure your apps and files are preserved. You can find instructions on how to do so at the [Reinstall Windows 10 page](https://support.microsoft.com/help/4000735/windows-10-reinstall).
+
+### Correct (SSH related) permission errors
+
+Like
```

Comment:
I'd recommend changing the wording here to:

`If you're seeing this error:`

Reply:
updated